examples/kubernetes/README.md

```diff
@@ -0,0 +1,235 @@
+# Using the Cloud SQL proxy on Kubernetes
+
+The Cloud SQL proxy is the recommended way to connect to Cloud SQL, even when
+using private IP. This is because the proxy provides strong encryption and 
+authentication using IAM, which can help keep your database secure. 
+
+## Configure your application with Secrets
+
+In Kubernetes, [Secrets][ksa-secret] are a secure way to pass configuration 
+details to your application. You can create a Secret with details such as your
+database name, user, and password which can be injected into your application as
```

Comment:
Clarification... is a Secret considered a bundle of key/val pairs? Or is each key/val pair a "Secret"? I haven't looked at the Secrets documentation to know how it's broken down. Am I creating a single "Secret" which contains db name, db user and db password? Or is each one of those called a "Secret" that is then bundled together somehow?

Reply:
Added a line clarifying a single Secret is composed of multiple key/values.